lib/elixir/test/elixir/kernel/errors_test.exs

```diff
@@ -706,7 +706,7 @@ defmodule Kernel.ErrorsTest do
 
   test "invalid macro" do
     assert_eval_raise CompileError,
-                      "nofile: invalid quoted expression: {:foo, :bar, :baz, :bat}",
+                      "nofile: invalid quoted expression: {:foo, :bar, :baz, :bat}\nYou may have meant to escape this with Macro.escape/1",
```

Comment:
I would propose for us to use a regex so we don't have to change this every time. Something like this:

```suggestion
                      ~r"nofile: invalid quoted expression: {:foo, :bar, :baz, :bat}",
```